polars/polars-time/src/chunkedarray/utf8/patterns.rs

```diff
@@ -94,10 +87,6 @@ pub(super) static DATETIME_Y_M_D: &[&str] = &[
     // 2021/12/31 11:54 PM
     "%Y/%m/%d %I:%M %p",
     "%Y-%m-%d %I:%M %p",
-    // --
```

Comment:
Was this removed by accident?

Reply:
I wasn't really sure what the pattern of comment separators was, but I've put it back now